apps/test/unit/lib/levelbuilder/lesson-editor/activitiesTestData.js

```diff
@@ -1,11 +1,13 @@
 export const sampleActivities = [
   {
+    id: 11,
     key: 'activity-1',
```

Comment:
This activity data has keys in it. If we are removing key and using id instead we should consider removing key from the test data.

Reply:
we are still using `key` for activity and activity section in the lesson editor. we are not using `key` in the teacher-facing lesson overview page. so I don't think I can remove it yet. I've got a to-do item to figure out if these two things should really be sharing the same shape. since they have different fields it seems like they should actually be using different shapes.